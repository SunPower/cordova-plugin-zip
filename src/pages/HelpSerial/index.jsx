import React from 'react'
import clsx from 'clsx'
import ModalLayout from '../../components/ModalLayout'
import ModalItem from '../../components/ModalItem'
import paths from '../Router/paths'
import './HelpSerial.scss'

function HelpSerial({ history, className }) {
  const classes = clsx('help-serial', className)
  return (
    <ModalLayout
      className={classes}
      history={history}
      title="Device serial number"
      from={paths.SIGNUP_SERIAL}
    >
      <ModalItem
        overtitle="We need a monitoring device serial number to create your account"
        title="You may have received this:"
      >
        <ul>
          <li>In a welcome email from sunpower</li>
          <li>add more</li>
        </ul>
      </ModalItem>
    </ModalLayout>
  )
}

export default HelpSerial
