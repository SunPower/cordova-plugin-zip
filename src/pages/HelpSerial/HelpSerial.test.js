import React from 'react'
import { shallow } from 'enzyme'
import HelpSerial from '.'

describe('HelpSerial page', () => {
  it('renders without crashing', () => {
    const component = shallow(<HelpSerial />)
    expect(component).toMatchSnapshot()
  })
})
