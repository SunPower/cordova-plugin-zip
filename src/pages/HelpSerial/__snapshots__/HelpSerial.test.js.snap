// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelpSerial page renders without crashing 1`] = `
<ModalLayout
  className="help-serial"
  from="/sign-up/serial"
  title="Device serial number"
>
  <ModalItem
    overtitle="We need a monitoring device serial number to create your account"
    title="You may have received this:"
  >
    <ul>
      <li>
        In a welcome email from sunpower
      </li>
      <li>
        add more
      </li>
    </ul>
  </ModalItem>
</ModalLayout>
`;
